code/game/turfs/flooring/flooring_premade.dm

```diff
@@ -242,7 +242,10 @@
 	if(ishuman(A))
 		var/mob/living/carbon/human/L = A
 		var/turf/Above = GetAbove(src)
+		if(L.isChild() || L.job == "Red Sniper" || L.job == "Blue Sniper") // No species for manlets, oversight?
```

Comment:
Child is fine, snipers is not.

Reply:
Reverted snipers being manlets, we're al good.